scripts/sdt.coffee

```diff
@@ -1,46 +1,152 @@
-class Schedule
-  constructor: (@talks)->
-  toString: ->
-    @talks.map((talk)->
-      "- *#{talk.title}* _by #{talk.speaker.name}_"
-    ).join("\n")
+# Description:
```

Comment:
I think for block comments you can use use something like this:
###
your multiline comment 
here
###